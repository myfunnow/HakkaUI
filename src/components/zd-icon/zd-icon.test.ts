
import { describe, it, expect } from "vitest"
import { mount } from '@vue/test-utils'
import ZdIcon from './zd-icon.vue'

describe('Icon', async () => {
  expect(ZdIcon).toBeTruthy()

  const wrapper = mount(ZdIcon, {

  })

  it('should to match the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('should ...', () => {
    
  })
})
