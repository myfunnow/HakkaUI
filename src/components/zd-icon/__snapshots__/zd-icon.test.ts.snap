// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Icon > should to match the snapshot 1`] = `
"<div class="zd-icon flex items-center justify-center transition-opacity">
  <!---->
</div>"
`;
